mmpose/datasets/datasets/face/face_aflw_dataset.py

```diff
@@ -0,0 +1,209 @@
+import os
+from collections import OrderedDict
+
+import json_tricks as json
+import numpy as np
+
+from mmpose.core.evaluation.top_down_eval import keypoint_nme
+from mmpose.datasets.builder import DATASETS
+from .face_base_dataset import FaceBaseDataset
+
+
+@DATASETS.register_module()
+class FaceAFLWDataset(FaceBaseDataset):
+    """Face AFLW dataset for top-down face keypoint localization.
+
+    `Annotated Facial Landmarks in the Wild: A Large-scale,
+    Real-world Database for Facial Landmark Localization.
+    In Proc. First IEEE International Workshop on Benchmarking
+    Facial Image Analysis Technologies, 2011`
+
+    The dataset loads raw images and apply specified transforms
+    to return a dict containing the image tensors and other information.
+
+    The landmark annotations follow the 68 points mark-up.
```

Comment:
Where to find the `68 points mark-up`? How about adding a docs page listing all keypoint formats?

Reply:
It is hard to explain the keypoint formats in words. Images will be helpful to show the keypoint formats.

Reply:
Okay, any "official" link to this keypoint format?